Src/BootCamp.Chapter/Challenge.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class Challenge
+    {
+        public static string LongestSubstring(string s)
+        {
+            if (s is null)
+            {
+                throw new ArgumentNullException(nameof(s));
+            }
+
+            var charArray = s.ToLower().ToCharArray();
+            string returnString = "";
+
+            foreach (char character in charArray)
+            {
+                if (!returnString.Contains(character))
```

Comment:
The solution is completely wrong.
Your task is not to return all unique characters. Your task is to return the longest word in a string, that has no repeating characters.
For example:
"abcaaaklm"
answer is "aklm" and not "abcklm"

Reply:
Next time please run the solution in their system- it will fail and then fix it, adding the sample input that fails.